src/dataframe/dataframe.jl

```diff
@@ -1024,15 +930,23 @@ select!(df::DataFrame, c::Any) = select!(df, index(df)[c])
 
 """
     select(df::DataFrame, inds, copycols::Bool=true)
+    select(dfv::SubDataFrame, inds, copycols::Bool=true)
+    select(dfr::DataFrameRow, inds)
 
-Create a new `DataFrame` that contains columns from `df` specified by `inds` and return it.
```

Comment:
Better keep a general description of what the function does, disregarding the particular `AbstractDataFrame` type. The last sentence about `inds` can probably go here too.

Reply:
OK